File: 백준/Bronze/10869. 사칙연산/사칙연산.js
input = require('fs').readFileSync('/dev/stdin').toString().trim().split(' ');

const A = parseInt(input[0], 10);
const B = parseInt(input[1], 10);

const result1 = A + B;
const result2 = A - B;
const result3 = A * B;
const result4 = parseInt(A / B);
const result5 = A % B;

console.log(result1);
console.log(result2);
console.log(result3);
console.log(result4);
console.log(result5);

File: 백준/Bronze/10869. 사칙연산/사칙연산.kt
fun main() {
    val input = readln().trim().split(" ").map{ it.toInt() }
    
    val A = input[0]
    val B = input[1]
    
    println(A + B)
    println(A - B)
    println(A * B)
    println(A / B)
    println(A % B)
}